moveit_core/utils/CMakeLists.txt

```diff
@@ -1,35 +1,30 @@
 set(MOVEIT_LIB_NAME moveit_utils)
 
+find_package(moveit_resources REQUIRED)
```

Comment:
Lifting moveit_resources to the normal build, requires an appropriate dependency change in `package.xml` - one reason, why we decided to build this lib in testing mode only.

Reply:
Good point. 

It seems to me that elevating moveit_resources to a normal dependency is inevitable if we want to be able to use libmoveit_test_utils outside of the moveit_core package. At that point, I'm not entirely sure we need moveit_test_utils to be separate from moveit_utils.

Reply:
libmoveit_test_utils should be definitely different from moveit_utils, as it depends on gtest!
One option to avoid the `moveit_resources` dependency, is to find another way (e.g. using `rospack`) to find the path to the `moveit_resources` package.

Reply:
The most recent commit keeps moveit_resources as a test_depend and replaces `MOVEIT_TEST_RESOURCES_DIR` with `ros:￼:getPath(moveit_resources)`

Reply:
Great work! If you replaced all occurrences of `MOVEIT_TEST_RESOURCES_DIR`, in future we might want to remove the corresponding [generator code from cmake](https://github.com/ros-planning/moveit_resources/blob/02b47539c1062d4bb2c790ea11a333372718cee7/CMakeLists.txt#L9-L15) as well, which I always considered clumsy.